code/modules/antagonists/fugitive/hunter.dm

```diff
@@ -55,6 +55,9 @@
 /datum/antagonist/fugitive_hunter/get_team()
 	return hunter_team
 
+/datum/antagonist/fugitive_hunter/apply_innate_effects()
+	add_team_hud(owner.current)
```

Comment:
```suggestion
/datum/antagonist/fugitive_hunter/apply_innate_effect(mob/living/mob_override)
	add_team_hud(mob_override || owner.current)
```

There's some cases (on body transfer) that needs to take into account a mob override.